src/display/editor/editor.js

```diff
@@ -221,6 +225,7 @@ class AnnotationEditor {
     this.div.className = this.name;
     this.div.setAttribute("id", this.id);
     this.div.tabIndex = 0;
+    this.div.style.zIndex = this.#zIndex;
```

Comment:
Do we really need *both* the static and the private member?
Could we not simply do the following instead:
```suggestion
    this.div.style.zIndex = AnnotationEditor._zIndex++
```